src/css/graphql/graphql-playground.css

```diff
@@ -339,6 +339,97 @@ div[data-radix-popper-content-wrapper] * {
     color: var(--base-text-color);
 }
 
+/*
+    Removes the scroll from the GraphiQL Explorer plugin, since the parent container
+    with the class "graphiql-plugin" already provides vertical (y-axis) scrolling.
+    For horizontal (x-axis) scrolling, each GraphQL queries/mutations container will handle its own scroll.
+*/
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div {
+    overflow: hidden !important;
+}
+
+/*
+  Adds horizontal (x-axis) scrolling to each GraphQL queries/mutations container.
+*/
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:first-child > div,
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:has(.variable-editor-title) {
+    overflow-x: auto !important;
+}
+
+/* Added a separation line between the container with queries/mutations and the container with the form for adding a new query/mutation. */
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:first-child {
+    border-bottom: 1px solid rgb(214, 214, 214);
+}
+
+/* Removes the top border from the form for adding a new query/mutation,
+   which was used as a visual horizontal separator between the queries/mutations and the form. */
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:last-child .variable-editor-title {
+    border: none !important;
+    padding: 0 !important;
+}
+
+.graphql-playground-view #graphiql .docExplorerWrap .graphiql-explorer-root > div:has(.variable-editor-title) {
+    margin-top: 16px;
+}
+
+/* =========================*/
+/*     Button overrides     */
+/* =========================*/
+.graphql-playground-view #graphiql .docExplorerWrap button.toolbar-button {
```

Comment:
Remove the tag names from selectors where there is a class or id already.

Reply:
done